dashboard/app/views/layouts/_header.html.haml

```diff
@@ -103,7 +103,7 @@
 
 - if show_header2
   - if request.path == "/home" || request.path == "/courses"
-    - if user_type == "teacher"
+    - if user_type == "teacher" || !user_type
```

Comment:
possibly `user_type` != `student` I think.  But worth testing and only if you think that's more readable.